Remotion/Validation/UnitTests/RoleCollectors/RemovingObjectValidationRuleCollectorTest.cs

```diff
@@ -111,19 +112,30 @@ public void RegisterValidator_WithMultipleValidators_AddsAllValidatorsToCollecto
     }
 
     [Test]
-    [Ignore("TODO RM-5906")]
     public void RegisterValidator_ValidatorTypeDoesNotImplementIObjectValidator_ThrowsArgumentException ()
     {
+      Assert.That(
+          () => _removingObjectValidationRuleCollector.RegisterValidator(typeof(Customer), typeof(CustomerValidationRuleCollector1), null),
+          Throws.ArgumentException
+              .With.ArgumentExceptionMessageEqualTo("Parameter 'validatorType' is a 'Remotion.Validation.UnitTests.TestDomain.Customer', "
+                                                    + "which cannot be assigned to type 'Remotion.Validation.Validators.IObjectValidator'.", "validatorType"));
```

Comment:
Please check other usages of this helper method. we typically format it this way:
```suggestion
              .With.ArgumentExceptionMessageEqualTo(
                  "Parameter 'validatorType' is a 'Remotion.Validation.UnitTests.TestDomain.Customer', which cannot be assigned to type 'Remotion.Validation.Validators.IObjectValidator'.",
                   "validatorType"));
```
If the message still is too longe, we do a line break where it's stylistically oportune (e.g. at a comma) like you did now. The important part is that the parameters start in a separate line and tehre's is just one parameter per line.

Reply:
I checked for other usages and there are some tests where it is actually not in the correct format as you mentioned. Should I create an issue and clean that up too or is it not necessary for now?

Reply:
yeah, the older ones. Anything new is clean :) if you want, yeah, create an issue, thanks. We can probably find them via regex search rather easily. Cleanup's not really an urgent issue. More like "when there's work on that test" or "there's formatting stuff that touches this" it gets fixed, too. Or someone thinks it's just an hour or two and there's a quick win and some free time.

Reply:
https://re-motion.atlassian.net/browse/RM-8847